src/Maths/Silk.NET.Maths/Simd/Simd128.Divide.cs

```diff
@@ -0,0 +1,88 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if INTRINSICS
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+#if SSE
+using System.Runtime.Intrinsics.X86;
+#endif
+#if AdvSIMD
+using System.Runtime.Intrinsics.Arm;
+#endif
+
+namespace Silk.NET.Maths
+{
+    public static unsafe partial class Simd128
+    {
+        /// <summary>
+        /// Performs hardware-accelerated Divide on 128-bit vectors.
+        /// </summary>
+        [MethodImpl(Scalar.MaxOpt)]
+        public static Vector128<T> Divide<T>(Vector128<T> left, Vector128<T> right) where T : unmanaged
+        {
+
```

Comment:
Consider adding a TODO and raising an issue that non-floating point types need to be added here.

Reply:
Might be sensible to have a general SIMD tracking issue as there'll probably be lots of notes that we'll want to keep track of.

Reply:
Seems fair to me as well